lockable/provider_http.py

```diff
@@ -13,18 +16,43 @@ class ProviderHttp(Provider):
 
     def __init__(self, uri: str):
         """ ProviderHttp constructor """
+        self._configure_http_strategy(uri)
         super().__init__(uri)
 
+    def _configure_http_strategy(self, uri):
+        """ configure http Strategy """
+        retry_strategy = Retry(
+            total=5,
+            redirect=5,
+            status_forcelist=[
```

Comment:
What about the case where there is no server at all?

Reply:
added `connect` parameter that retries connection errors.